plugins/logs/plugin.go

```diff
@@ -56,14 +62,26 @@ type ReportingConfig struct {
 
 // Config represents the plugin configuration.
 type Config struct {
+	Plugin        *string         `json:"plugin"`
```

Comment:
Update the configuration reference doc for Decision Logs with the new `plugin` field.

Reply:
Nice catch. Done.